resources/original_messages.json

```diff
@@ -424,7 +424,7 @@
     "desciption": "An error message shown when the user tries to install Outline backend services but failed for some reason."
   },
   "outline_services_installed": {
-    "message": "Outline has been successfully initialized, please try again to connect to the server.",
+    "message": "Outline has been successfully initialized, please restart your machine for changes to take effect.",
```

Comment:
We need to restart the machine because Linux does not provide a way to refresh new group memberships of the current user.  [Docker also requires to do so.](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)

Reply:
You just need to log out, then and log back in.

In any case, that's a poor user experience. Can't you use `newgroup outlinevpn` or `sg` somehow?

https://linux.die.net/man/1/newgrp
https://linux.die.net/man/1/sg

Reply:
I agree the user experience is not good. But it seems we don't have a perfect solution now because:

1. [`newgrp`](https://linux.die.net/man/1/newgrp) will only replace the current shell. If we run it in installation script, it just replaces the root shell; if we run it in node (`child_process.exec`), it only replaces the shell in sub-process (not the parent electron process), neither will update the group of current electron process.
2. [`sg`](https://linux.die.net/man/1/sg) works when you launch the app: `sg outlinevpn ./Outline-Client.AppImage`. Although we can do a relaunch after installation, what if the user closes the app and launches it again manually (e.g., double click the AppImage file), we don't have a way to inject the group (because the user did not restart the machine yet).
3. We can always wrap the `sg outlinevpn ./Outline-Client.AppImage` in the entry point of AppImage, but that requires additional efforts to investigate how to get rid of electron builder and re-package the app.
4. Linux provides [`setgid()`](https://www.freebsd.org/cgi/man.cgi?query=setegid&sektion=2&n=1) function (which is similar to what `sg` is doing) which we can call it in electron process, but that requires the go-plugin to be implemented first.

So in this case, I think we can update the message first, and implement some methods I mentioned above to improve the user experience in the future.

P.S.: log-out would be good for most desktop apps, but not VMWare virtual machines (which requires a full restart). So to be safe, I updated the message to "restart" instead of "log out". ([Docker's installation step also mentions about it.](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user))